pom.xml

```diff
@@ -567,6 +569,8 @@ limitations under the License.]]>
                 <!-- Present at the root of each module that doesn't skip this goal -->
                 <analysisConfigurationFile>revapi.json</analysisConfigurationFile>
               </analysisConfigurationFiles>
+              <!-- Prevents query-builder from re-checking core. -->
+              <checkDependencies>false</checkDependencies>
```

Comment:
I have mixed feelings about this, because it also excludes all external dependencies (Netty, etc). On the one hand, checking external API changes should not be our concern, on the other it would be good to know when upgrading a dependency breaks things.

If we remove this, core is checked again by the query-builder build. So we need to either:
- duplicate all the ignores of core's `revapi.json` in query-builder's
- use a single `revapi.json` at the parent level, but that brings the question on how to access it from the child modules: I don't like relative paths, it goes against the principles of Maven.